### PR TITLE
refs #228 : rss feed on articles

### PR DESCRIPTION
**Pull request by mpetitdant**

### CODE REVIEW

File: server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java [hunk 1]
```diff
@@ -25,11 +32,11 @@
 @RequestMapping("/api/article")
 public class ArticleController {
 
-    @Autowired
-    private ArticleRepository articleRepository;
+    public static final String MIX_IT_ARTICLE = "https://www.mix-it.fr/article/";
+    public static final String MIX_IT_ARTICLES = "https://www.mix-it.fr/articles";
```
Comment (javamind) [inline server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java]: You should inject ```AbsoluteUrlFactory``` to find the baseurl (https://www.mix-it.fr/)

File: server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java [hunk 2]
```diff
@@ -51,6 +58,43 @@
                 .body(result);
     }
 
+    @RequestMapping("/rss/fr_FR")
+    @ApiOperation(value = "Articles RSS (fr)", httpMethod = "GET")
```
Comment (javamind) [inline server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java]: I used i leaflet controller /fr and /en

File: server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java [hunk 3]
```diff
@@ -51,6 +58,43 @@
...
+    @RequestMapping("/rss/us_US")
+    @ApiOperation(value = "Articles RSS (en)", httpMethod = "GET")
+    @JsonView(FlatView.class)
+    public String getArticlesRssEn() throws IOException, FeedException {
+        return getArticlesFeed(SessionLanguage.en);
+    }
+
+    @JsonView(FlatView.class)
+    public String getArticlesFeed(SessionLanguage language) throws IOException, FeedException {
+        SyndFeed feed = new SyndFeedImpl();
```
Comment (javamind) [inline server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java]: why do you need to annotate this internal method with ```java @JsonView(FlatView.class)```
Your method can be set with a protected visibility

File: server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java (hunk 2 above)
Comment (mpetitdant) [inline server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java]: I used the values returned by ctrl.currentLanguage in view, quick & dirty way. We can check that together today.

File: server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java (hunk 1 above)
Comment (mpetitdant) [inline server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java]: Done, thanks for the tip

File: server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java (hunk 2 above)
Comment (mpetitdant) [inline server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java]: As seen today, I felt that as is.

File: server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java (hunk 3 above)
Comment (mpetitdant) [inline server/src/main/java/org/mixit/cesar/site/web/api/ArticleController.java]: Corrected

**Pull request merged**